README.md

```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
 The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
-## Advanced Configuration
+### Advanced Configuration
 
 To fully configure the Helm chart, see the [advanced
 configuration](docs/advanced-configuration.md).
 
+## Setup Splunk
```

Comment:
this seems redundant. you already mentioned in pre req with links. I say remove.

Reply:
removed